CMakeLists.txt

```diff
@@ -60,16 +60,28 @@ option( DETRAY_SVG_DISPLAY "Build ActSVG display module" OFF)
 option( DETRAY_BUILD_SYCL "Build the SYCL sources included in detray" OFF )
 option( DETRAY_BUILD_CUDA "Build the CUDA sources included in detray"
    ${DETRAY_BUILD_CUDA_DEFAULT} )
-option( DETRAY_BUILD_TESTING "Build the (unit) tests of Detray"
-   TRUE )
-cmake_dependent_option( DETRAY_BENCHMARKS "Enable benchmark tests" TRUE
-   "DETRAY_BUILD_TESTING" OFF )
-option( DETRAY_BUILD_TUTORIALS "Build the tutorial executables of Detray"
+option( DETRAY_BUILD_TEST_UTILS "Build the test utility library of Detray"
    ON )
+option( DETRAY_BUILD_UNITTESTS "Build the unit tests of Detray" ON )
+option( DETRAY_BUILD_INTEGRATIONTESTS "Build the integration tests of Detray" 
+   ON )
+option( DETRAY_BUILD_BENCHMARKS "Build the benchmark tests" ON  )
```

Comment:
I think it might be nice to have these OFF by default!

Reply:
Sure! Do I need to switch ```DETRAY_BUILD_BENCHMARKS``` to ```ON``` somewhere for the continuous benchmark @beomki-yeo ?

Reply:
It is OK to turn it off by default but please make sure that it is built in the detray CI at least.

The benchmark CI will also need to be fixed as it seems changing the name of options

Reply:
Not changing the option name would be the best

Reply:
I harmonized the naming of that parameter now and opened an MR into the benchmark repo. This also allows to switch off the building of the unit and integration tests there.